cli/bin/locator-path.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+
+const relativePath = '_build/install/default/lib/grain/stdlib';
+const absolutePath = path.resolve(__dirname, '../../', relativePath);
+
+module.exports = absolutePath;
```

Comment:
I was having some issues with the `grain` command not being able to find `_build/install/default/lib/grain/stdlib` based off of where it was executed from. I created this file to ensure we are always pointing towards an absolute path for `stdlib`.

 I then required this file from both `run.js` and `compile.js`.

Reply:
Yeah, the `grain` CLI tool on `master` has some bitrot. A somewhat more "legitimate" fix was implemented last night in e1578e6 (on the `gc` branch), but wiring up `grain` to `GRAIN_ROOT` is still a bit of a work in progress...